Sources/Domain/TwilioVerifySNA/TwilioVerifySNASession.swift

```diff
@@ -158,7 +157,8 @@ final class TwilioVerifySNASession: TwilioVerifySNA {
         }
 
         requestManager.processSNAURL(
-            url
+            url,
+            using: .any
```

Comment:
Should we expose a way to configure the version?

Reply:
I will remove this part, the idea is to alway use `.any`, since it allows to try both ip versions automatically.